interface/src/Application.cpp

```diff
@@ -1296,7 +1295,7 @@ void Application::initializeUi() {
     rootContext->setContextProperty("Quat", new Quat());
     rootContext->setContextProperty("Vec3", new Vec3());
     rootContext->setContextProperty("Uuid", new ScriptUUID());
-    rootContext->setContextProperty("Assets", DependencyManager::get<AssetMappingsScriptingInterface>().data());
+    rootContext->setContextProperty("Assets", new AssetMappingsScriptingInterface());
```

Comment:
There was no need for this since it ended up being only in the QMLEngine.
I made the change be cause I thought I had to pass in the JSEngine on construction so we could call the callback with their context. Turns out you can't get that context with JSEngine.